Yoda/test.cc

```diff
@@ -59,8 +64,15 @@ TEST(Yoda, WritesToFile) {
   while (bricks::FileSystem::GetFileSize(persistence_file_name) != yoda_golden_data.size()) {
     ;  // Spin lock.
   }
-
   EXPECT_EQ(yoda_golden_data, bricks::FileSystem::ReadFileAsString(persistence_file_name));
+
+  api.Transaction([](PersistingAPI::T_DATA data) {
+    Dictionary<YodaEntryToPersist>::Mutator(data).Delete("one");
+  }).Wait();
+  while (bricks::FileSystem::GetFileSize(persistence_file_name) != yoda_golden_data_after_delete.size()) {
```

Comment:
P.S. If the persistence is broken (actual does not match golden), the test will hang indefinitely. Maybe compare `> 0`?